client/dashboard/sites/site/index.tsx

```diff
@@ -22,13 +23,18 @@ function Site() {
 	return (
 		<>
 			<HeaderBar>
-				<HStack justify={ isDesktop ? 'flex-start' : 'space-between' } spacing={ 4 }>
+				<HStack justify={ isDesktop ? 'flex-start' : 'space-between' } spacing={ 3 }>
 					<HeaderBar.Title>
 						<Dropdown
 							renderToggle={ ( { onToggle } ) => (
-								<Button className="dashboard-menu__item active" onClick={ () => onToggle() }>
+								<Button
+									className="dashboard-menu__item active"
+									icon={ chevronDownSmall }
```

Comment:
I would have expected the chevron to flip when the site selector is expanded.

But I guess we don’t do that in core components?

https://github.com/user-attachments/assets/bc936b8e-1791-4a18-8d8a-deb3206aab7b

Reply:
Yeah, I left it in the down position to mirror what we have elsewhere.